### PR TITLE
T3W1 PCB bring-up

### PR DESCRIPTION
**Pull request by TychoVrahe**
This PR adds RGB LED driver for T3W1. Syscall is added to enable driving from app. 

Old RGB LED driver is removed, as it is no longer needed after ditching old boards.

Power button configuration is added to rev A board. 

There was an error in setting the U5 clock with 32MHz HSE, so i fixed it here too. Some unused clock-configuration stuff is removed from startup_init.

Another HSE implications are for systick calculation, which need proper value set in stm32u5xx_hal_conf.h, so i replaced the HSE_xxMHz with native HSE_VALUE=xxx and move it to global. 

USB driver was adjusted to work with 32MHz HSE too.

SBU pins are configured for T3W1. I removed "sbu" from features wanted in firmware, kernel and unix, its only needed for prodtest.

Another MPU bugfix: kernel SRAM setting was off, as SRAM1 now contains framebuffe on U5G.

The mock display driver was modified a bit so it doesn't crash when someone tries some rendering - one framebuffer is allocated for that purpose.

### CODE REVIEW

File: core/embed/projects/bootloader/main.c [hunk 1]
```diff
@@ -369,7 +369,7 @@ int bootloader_main(void) {
 #ifdef USE_TOUCH
   secbool touch_initialized = secfalse;
   secbool allow_touchless_mode = secfalse;
-#ifdef TREZOR_MODEL_T3T1
+#if defined TREZOR_MODEL_T3T1 || defined TREZOR_MODEL_T3W1
   // on T3T1, tester needs to run without touch, so making an exception
```
Comment (prusnak) [inline core/embed/projects/bootloader/main.c:373]: maybe add T3W1 to the comment too?

Comment (TychoVrahe) [reply core/embed/projects/bootloader/main.c]: fixed https://github.com/trezor/trezor-firmware/pull/4350/commits/c927e916b21cd8a7ad65303ca8ddb0a8cb27c3e5

**Review by cepetr — CHANGES_REQUESTED** [review]

File: core/embed/io/display/stm32u5a9j-dk/display_fb.c [hunk 2]
```diff
@@ -41,8 +41,7 @@ uint8_t physical_frame_buffer_1[PHYSICAL_FRAME_BUFFER_SIZE];
 
 // The current frame buffer selector at fixed memory address
 // It's shared between bootloaders and the firmware
-__attribute__((section(".framebuffer_select"))) uint32_t current_frame_buffer =
-    0;
+uint32_t current_frame_buffer = 0;
```
[the remark this answers is not in the document]
Comment (TychoVrahe) [reply core/embed/io/display/stm32u5a9j-dk/display_fb.c]: fixed https://github.com/trezor/trezor-firmware/pull/4350/commits/10e3635d5406427d797a4236d916323efecc2859

File: core/embed/sys/mpu/stm32u5/mpu.c [hunk 3]
```diff
@@ -310,15 +314,17 @@ mpu_mode_t mpu_get_mode(void) {
...
   if (!drv->initialized) {
     return;
   }
 
-  drv->unpriv_fb_addr = (uint32_t)addr;
-  drv->unpriv_fb_size = size;
+  drv->active_fb_addr = (uint32_t)addr;
+  drv->active_fb_size = size;
+
+  mpu_reconfig(drv->mode);
```
[the remark this answers is not in the document]
Comment (TychoVrahe) [reply core/embed/sys/mpu/stm32u5/mpu.c:331]: fixed https://github.com/trezor/trezor-firmware/pull/4350/commits/6b3babcc6c70e8fcbd08128cee1341ead7d74e1c

File: core/embed/io/display/st-7789/display_fb.c [hunk 4]
```diff
@@ -208,12 +210,15 @@ static void copy_fb_to_display(uint8_t index) {
   uint16_t *fb = (uint16_t *)get_fb_ptr(index);
 
   if (fb != NULL) {
+    mpu_set_active_fb(fb, PHYSICAL_FRAME_BUFFER_SIZE);
```
[the remark this answers is not in the document]
Comment (TychoVrahe) [reply core/embed/io/display/st-7789/display_fb.c:213]: as discussed in person, lets postpone fixing this until needed

**Comment by TychoVrahe** [comment]
Added one more commit separating the bootargs from kernel/aux SRAM, adding separate MPU mode for handling it: https://github.com/trezor/trezor-firmware/pull/4350/commits/0be697c890acc1ece8b6f74374f1d693eafcea22, @cepetr please review that as well

File: core/embed/sys/startup/stm32/bootutils.c [hunk 5]
```diff
@@ -202,7 +212,7 @@ void __attribute__((noreturn)) secure_shutdown(void) {
 }
 
 void ensure_compatible_settings(void) {
-#ifdef TREZOR_MODEL_T
+#ifdef TREZOR_MODEL_To
```
Comment (cepetr) [inline core/embed/sys/startup/stm32/bootutils.c:215]: TREZOR_MODEL_To

**Review by cepetr — CHANGES_REQUESTED** [review]

File: core/embed/sys/mpu/stm32u5/mpu.c (hunk 3 above)
[the remark this answers is not in the document]
Comment (cepetr) [reply core/embed/sys/mpu/stm32u5/mpu.c:313]: extending the lock to `mpu_reconfig()` is not necessary. Locking writes to variables seems sufficient in this case.

File: core/embed/sys/startup/stm32/bootutils.c (hunk 5 above)
Comment (TychoVrahe) [reply core/embed/sys/startup/stm32/bootutils.c]: fixed https://github.com/trezor/trezor-firmware/commit/9516ebb75aad17aee025513f114929e197ac02f8

File: core/embed/sys/mpu/stm32u5/mpu.c (hunk 3 above)
[the remark this answers is not in the document]
Comment (TychoVrahe) [reply core/embed/sys/mpu/stm32u5/mpu.c:315]: fixed https://github.com/trezor/trezor-firmware/pull/4350/commits/53fcc2b028a246d9ae080947c572ccba44941563

**Review by cepetr — APPROVED** [review]
I went carefully through all these memory layout and MPU changes, and they look good to me.

**Pull request merged**